src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -766,7 +766,9 @@
     </COMReference>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="$(MicrosoftExtensionsCachingMemoryVersion)"  />
+    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="$(MicrosoftExtensionsCachingMemoryVersion)" />
+    <PackageReference Include="Microsoft.Identity.Client.Broker" Version="4.64.0" />
```

Comment:
[M.I.C.Broker](https://learn.microsoft.com/en-us/dotnet/api/microsoft.identity.client.broker?view=msal-dotnet-latest) is not available for net6-windows, .Netframwork and legacy application.

Reply:
@mdaigle we wanna conditionally include this package dependency, based on environment/config it's available in.

Secondly, please consider including this in nuspec as well.